tests/resources/test_project.py

```diff
@@ -211,3 +374,624 @@ def test_get_projects_pagination(self):
+                        "cohortPatients": [
+                            {
+                                "patient": {
+                                    "id": "patient-1"
+                                },
+                                "metricList": {
+                                    "metrics": [
+                                        {
+                                            "id": "metric-1",
+                                            "updated_at": 1673467628.837126,
+                                            "created_at": 1673467628.837126,
+                                            "type": "LAST_UPLOAD",
+                                            "data_type": "APPLEWATCH_TREMOR",
+                                            "value": 1648235452.965804,
+                                            "time_interval": "FOURTEEN_DAYS"
+                                        },
+                                    ]
+                                },
+                                "code_name": "code name 1",
+                                "created_at": 1673467625.063822,
+                                "updated_at": 1673467625.063822,
+                                "created_by": "user 1",
+                                "updated_by": "user 2",
+                            },
+                        ],
+                        "pageInfo": {
+                            "codeNameEndCursor": "code name 1"
+                        },
+                    }
+                }
+            },
+            {
+                "cohort": {
+                    "cohortPatientList": {
+                        "cohortPatients": [
+                            {
+                                "patient": {
+                                    "id": "patient-1"
+                                },
+                                "metricList": {
+                                    "metrics": [
+                                        {
+                                            "id": "metric-1",
+                                            "updated_at": 1673467628.837126,
+                                            "created_at": 1673467628.837126,
+                                            "type": "LAST_UPLOAD",
+                                            "data_type": "APPLEWATCH_TREMOR",
+                                            "value": 1648235452.965804,
+                                            "time_interval": "FOURTEEN_DAYS"
+                                        },
+                                    ]
+                                },
+                                "code_name": "code name 1",
+                                "created_at": 1673467625.063822,
+                                "updated_at": 1673467625.063822,
+                                "created_by": "user 1",
+                                "updated_by": "user 2",
+                            },
```

Comment:
To test this is paginating correctly, you shouldn't return patient 1 in the second page, only in the first page.

```suggestion
```